### PR TITLE
Enabled DR. CI for vision repo

### PR DESCRIPTION
**Pull request by DanilBaibak**
Enabled DR. CI for vision repo.

### CODE REVIEW

File: torchci/lib/bot/utils.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function isPyTorchOrg(owner: string): boolean {
 }
 
 export function isPyTorchPyTorch(owner: string, repo: string): boolean {
-  return isPyTorchOrg(owner) && repo === "pytorch";
+  return isPyTorchOrg(owner) && ["pytorch", "vision"].includes(repo);
 }
```
Comment (malfet) [inline torchci/lib/bot/utils.ts:17]: This is wrong, please add new function, say `isPyTorchCoreOrTorchvision` or `isPyTorchVision`

**Review by malfet — CHANGES_REQUESTED** [review]
`isPyTorchPyTorch` is used from more than one place in the codebase. Also, IMO, this creates a mismatch between function name and behavior. 
Please add a new function (say isPyTorchOrTorchvision`) or if we want to enable DrCI for all projects in PyTorch org, just use `isPyTorchOrg` as guard in drciUtils.ts

File: torchci/lib/bot/utils.ts (hunk 1 above)
Comment (seemethere) [reply torchci/lib/bot/utils.ts:17]: Alternatively we can just have a Dr. CI specific one.

`isDrCIEnabled`

Comment (DanilBaibak) [reply torchci/lib/bot/utils.ts:17]: I like the idea of having `isDrCIEnabled` 👍

File: torchci/lib/drciUtils.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import _ from "lodash";
 import { Octokit } from "octokit";
 import { IssueData } from "./types";
 import fetchIssuesByLabel from "lib/fetchIssuesByLabel";
-import { isPyTorchPyTorch } from "./bot/utils";
+import { isPyTorchPyTorch, isDrCIEnabled } from "./bot/utils";
```
Comment (ZainRizvi) [inline torchci/lib/drciUtils.ts:5]: nit 

```suggestion
import { isDrCIEnabled } from "./bot/utils";
```

**Review by ZainRizvi — APPROVED** [review]

**Review by seemethere — APPROVED** [review]

**Review by osalpekar — APPROVED** [review]
Awesome!

**Pull request merged**